Aniruddha/Script/README.md

```diff
@@ -44,86 +44,60 @@ create table SLOGANS (
 	end;
 */
 
-insert into SLOGANS (Company_Name, Founded_Date, Company_Slogan)
-values('3M', to_date('13-06-1902','dd-mm-yyyy'), 'Innovation');
 
-insert into SLOGANS (Company_Name, Founded_Date, Company_Slogan)
-values('Adobe', to_date('01-12-1982','dd-mm-yyyy'), 'Better by Adobe');
+ insert into SLOGANS values('3M', to_date('13-06-1902','dd-mm-yyyy'), 'Innovation');
```

Comment:
@shubham007kumar Kindly remove the **space** before `insert` keyword on line **48**.
commit hash 7ff98b3751b2bd18ba4a4eb51254d233d1dce780.